src/tests/roc_audio/test_depacketizer.cpp

```diff
@@ -300,74 +301,136 @@ TEST(depacketizer, overlapping_packets) {
     expect_output(dp, SamplesPerPacket / 2, 0.33f);
 }
 
-TEST(depacketizer, frame_flags_dropped_packets_end) {
+TEST(depacketizer, frame_flags_packet_drops) {
     packet::ConcurrentQueue queue(0, false);
     Depacketizer dp(queue, pcm_decoder, ChMask, false);
 
-    const packet::timestamp_t ts1 = SamplesPerPacket * 3;
-    const packet::timestamp_t ts2 = SamplesPerPacket * 2;
-    const packet::timestamp_t ts3 = SamplesPerPacket * 1;
+    packet::PacketPtr packets[] = {
+        new_packet(SamplesPerPacket * 3, 0.11f),
+        new_packet(SamplesPerPacket * 1, 0.11f),
+        new_packet(SamplesPerPacket * 2, 0.11f),
+        new_packet(SamplesPerPacket * 6, 0.11f),
+        new_packet(SamplesPerPacket * 2, 0.11f),
+        new_packet(SamplesPerPacket * 3, 0.11f),
+    };
 
-    queue.write(new_packet(ts1, 0.11f));
-    queue.write(new_packet(ts2, 0.22f));
-    queue.write(new_packet(ts3, 0.33f));
+    unsigned frame_flags[] = {
+        Frame::FlagFull | !(Frame::FlagPacketDrops),
```

Comment:
`!` is a logical operation, not a bitwise one. It returns either 0 or 1.

Reply:
BTW, GCC 7 even generates a warning (see failed build):

```
src/tests/roc_audio/test_depacketizer.cpp:318:51: error: 
   enum constant in boolean context [-Werror=int-in-bool-context]
         Frame::FlagFull | !(Frame::FlagPacketDrops),
```

Reply:
Damn..it's awful, I've fixed it.